ibm_catalog.json

```diff
@@ -220,13 +224,42 @@
               ]
             },
             {
-              "key": "tags"
+              "key": "name"
+            },
+            {
+              "key": "plan",
+              "options": [
+                {
+                  "displayname": "enterprise",
+                  "value": "enterprise"
+                },
+                {
+                  "displayname": "platinum",
+                  "value": "platinum"
+                }
+              ]
             },
             {
-              "key": "access_tags"
+              "key": "elasticsearch_resource_tags",
```

Comment:
do we have more `resource_tags` ? should be `resource_tags` only without `elasticsearch` since we have only one `resource_tags`